examples/firebase-auth-firestore/README.md

```diff
@@ -12,23 +12,34 @@ See the screen recording at `./screen_recording.gif` or Open this example on [Co
 
 ## Example
 
-To run it, you need to:
+To run it, you need to either:
 
-1. Create a Firebase Project
+### 1. Run against a Firebase Project
+
+1. [Create a Firebase Project](https://console.firebase.google.com)
 2. Enable Auth (with email) and Firestore
 3. Add a Web App
-4. Get the admin-sdk and client-sdk credentials
-5. Save them in the `.env`-file
+4. Get the [admin-sdk](https://firebase.google.com/docs/admin/setup#initialize-sdk) and [client-sdk credentials](https://firebase.google.com/docs/web/learn-more#config-object)
+5. Save them to SERVICE_ACCOUNT and CLIENT_CONFIG in the `.env`-file
+
+### 2. Use the Firebase emulators
+
+1. Run `yarn emulators` in one terminal window
+2. Run `yarn dev` in a second
+
+When the SERVICE_ACCOUNT and CLIENT_CONFIG environment variables have not been set, `yarn dev` will default to using the local emulator.
+
+When you run `yarn emulators`, an initial user is created with credentials user@example.com:password. This can be configured in `firebase-fixtures/auth/accounts.json` or via the emulator UI.
```

Comment:
```suggestion
When you run `npm run emulators`, an initial user is created with credentials `user@example.com:password`. This can be configured in `firebase-fixtures/auth/accounts.json` or via the emulator UI.
```